src/view.js

```diff
@@ -72,19 +72,15 @@ const View = Backbone.View.extend({
   },
 
   // Prepares the special `model` property of a view
-  // for being displayed in the template. By default
-  // we simply clone the attributes. Override this if
+  // for being displayed in the template. Override this if
   // you need a custom transformation for your view's model
   serializeModel() {
-    if (!this.model) { return {}; }
-    return _.clone(this.model.attributes);
+    return this.model.attributes;
   },
 
-  // Serialize a collection by cloning each of
-  // its model's attributes
+  // Serialize a collection
   serializeCollection() {
-    if (!this.collection) { return {}; }
-    return this.collection.map(function(model) { return _.clone(model.attributes); });
+    return _.map(this.collection.models, model => model.attributes);
```

Comment:
could do that, but this is faster.  don't care much either way.. I doubt this sees much use.